test/Experimental/Permutations-test.jl

```diff
@@ -0,0 +1,42 @@
+@testset "Examples.Permutations" begin
+  p = Oscar.Permutations.@perm (1,2)(3,4)(5,6)
```

Comment:
Make sure to `export @perm` (and also any other macros), then you can just write
```suggestion
  p = @perm (1,2)(3,4)(5,6)
```